web/app/templates/auth/email/reset_password.html

```diff
@@ -1,8 +1,8 @@
-<p>Dear {{ user.first_name }},</p>
-<p>To reset your password <a href="{{ url_for('auth.password_reset', token=token, _external=True) }}">click here</a>.</p>
-<p>Alternatively, you can paste the following link in your browser's address bar:</p>
+<p>Kära {{ user.first_name }},</p>
+<p>Vänligen <a href="{{ url_for('auth.password_reset', token=token, _external=True) }}">klicka här</a> för att återställa ditt lösenord.</p>
+<p>Alternativt så kan du klistra in den följande länken i webbläsarens adressfält:</p>
 <p>{{ url_for('auth.password_reset', token=token, _external=True) }}</p>
-<p>If you have not requested a password reset simply ignore this message.</p>
-<p>Sincerely,</p>
-<p>The Flasky Team</p>
-<p><small>Note: replies to this email address are not monitored.</small></p>
\ No newline at end of file
+<p>Om du inte vill byta lösenord kan du ignorera detta meddelande.</p>
+<p>Hälsningar,</p>
+<p>Chronos Development Team</p>
+<p><small>Observera: svar till denna e-postadress ej blir lästa.</small></p>
```

Comment:
This line should be "Observera: svar till denna e-postadress läses ej" or "Observera: svar till denna e-postadress blir ej lästa".